app/src/components/BaseHeader.vue

```diff
@@ -28,15 +28,15 @@
     <span>
       <!-- optional navigation to jump to next item in a list -->
       <router-link
-        v-if="nextPath"
+        v-if="nextPath && !isUnverified"
```

Comment:
If I build for polygon and directly navigate to `http://localhost:3000/#/dgrants/19` (an unverified grant) I see a page that just says "No grant selected". If I navigate to `http://localhost:3000/#/dgrants/19?unverified=true`, then the page loads. 

It seems a bit odd that you'd need to add that flag just to view a grant, so just want to confirm that's the desired behavior

Reply:
I think this is not desired - tested this is well with grant 9. On main, if we navigate to a non-whitelisted grant like 9 or 19 for example, the grant shows is still viewable which is what we want.

Reply:
@mk1r  + @mds1 pushed in a fix. could you recheck this now
Simplified it a bit